homeassistant/components/sensor/bme680.py

```diff
@@ -4,10 +4,14 @@
 Temperature, humidity, pressure and volitile gas support.
 Air Qaulity calucaltion based on humidity and volatile gas.
```

Comment:
"Air Quality calculation"

Reply:
Well spotted.

Fixed.

Reply:
Well, I should have looked more closely I hd 3 letters transposed and ended up with 3 commits instead of one as I pulled the trigger way to fast, sorry about that.